StringCalculatorKata/StringCalculator.cs

```diff
@@ -0,0 +1,90 @@
+﻿namespace StringCalculatorKata
+{
+    public static class StringCalculator
+    {
+        public static int Add(string numbers)
+        {
+            var values = new List<string>();
+
+            if (numbers.StartsWith("//"))
+            {
+                values = GetValuesWithCustomDelimiter(numbers);
+            }
+            else
+            {
+                values = numbers.Split(',', '\n')
+                                .ToList();
+            }
+
+            var intNumbers = GetIntegers(values);
+
+            if (HasNegatives(intNumbers))
+            {
+                throw new NegativesNotAllowedException(GetNegatives(intNumbers));
+            }
+
+            return SumOfIntegers(intNumbers);
+        }
+
+        private static List<int> GetIntegers(List<string> values)
+        {
+            var intNumbers = new List<int>();
+            int tempInteger;
+
+            for (int i = 0; i < values.Count; i++)
```

Comment:
use foreach here 
or, it would be great if u used linq here!

Reply:
Done